### ISSUE TITLE
Stopped working

### ISSUE THREAD

**Issue by Rodrigo-Ayam** [WorldLanguages/DoNotDiscard#6]
Hello it hasn't been working anymore, hope you still can fix it :/

**Comment by WorldLanguages** [comment]
@Rodrigo-Ayam Is this because of a change on Chrome version 100? Keep in mind this extension has been untouched for months.

**Comment by Rodrigo-Ayam** [comment]
Yes, I saw it hasn't been updated in a while, is there any change you still work on it? Must be cause of some chrome update probably yeah. Well at least reinstalled it and seems to be working again, will hope it still does cause it is so much necessary, chrome has been worst each day, wasn't for mine lots of extensions been used would have migrated to another browser already.

**Comment by WorldLanguages** [comment]
> Yes, I saw it hasn't been updated in a while, is there any change you still work on it?

@Rodrigo-Ayam The code for this extension is very simple, it just tells Chrome to set every tab as not autoDiscardable, there's no potential update I could make.

**Comment by sobitthen** [comment]
Either Chrome bypasses this extension or there is some interaction from users to make this work as I am unsure if I need to change anything in this Discards page mentioned at the Chrome Store or not. Auto Discardable has a Toggle option, I assume this is to turn off the extension, and in Actions there are 2 options, Urgent and Proactive.  I had hoped the Readme would tell us how to use, but it is only a sentence or 2 for me. Is this dead or am I missing some user action to make it work?
Facebook keeps refreshing page whenever open a tab or use an app on computer. 
So if you leave page to research something to make an informed comment you lose your place...
Any tips to use this or dead?

**Comment by evandrocoan** [comment]
The extension is working, it is marking tabs as not auto-discadable, but Google Chrome is ignoring this setting and is discarding tabs anyway: chrome://discards/

<img width="131" height="155" alt="Image" src="https://github.com/user-attachments/assets/b746fd63-8bc0-4f04-9f35-93035a3469b4" />
